receiver/windowsservicereceiver/metadata.yaml

```diff
@@ -0,0 +1,41 @@
+type: windowsservice
+
+status:
+  class: receiver
+  stability:
+    development: [metrics]
+  distributions: [contrib]
+  codeowners:
+    active: [pjanotti, shalper2]
+  unsupported_platforms: [darwin, linux]
+
+attributes:
+  windows.service.name:
+    description: The name of the windows Service being reported.
+    type: string
+  windows.service.startup_mode:
+    description: Startup mode of Windows Service
+    enum: [boot_start, system_start, auto_start, demand_start, disabled]
+    type: string
+  
+metrics:
+  windows.service:
```

Comment:
Is this metric name right? Shouldn't it be `windows.service.status`? I'd be partial to dropping the `windows` prefix from it & the attributes but that's just my preference, I think the name not having `status` feels incorrect.

Reply:
I like the idea of adding `status`. The `windows` prefix on the metric name seems reasonable since the metric is specific to Windows Services. However, it becomes redundant to add `windows.service` to the attributes, just `name` and `startup_mode` seems reasonable.

Reply:
Metric and attributes renamed per above comment d1e007217ce2287dd82655cb431edc20bf046702